src/app/errorReports/newErrorReportForm/ErrorReportText.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Header from '../../../components/header';
+import Button from '../../../components/button';
+import TextArea from '../../../components/textarea';
+import withErrorReportsForm from '../withErrorReportsForm';
+
+class ErrorReportText extends Component {
+  constructor() {
+    super();
+    this.onTextChange = this.onTextChange.bind(this);
+  }
+
+  onTextChange(event) {
+    this.props.onChange(event.target.value);
+  }
+
+  render() {
+    return (
+      <div>
+        <Header size="large" className="locationTitle">Error Report</Header>
+        <div className="mx-4 mb-3 text-left">
+          <div className="w-100 mr-5">
+            <Header size="medium" className="mb-3">
+              Please provide the correction below
+              <br />
+              (Please don&#39;t enter any private information)
+            </Header>
+          </div>
+        </div>
+        <div>
+          <div className="mx-4 mb-1">
+            <TextArea
+              value={this.props.errorReportText}
+              onChange={this.onTextChange}
+              minRows={7}
+              autoFocus
+              fluid
+              className="border" // flex-1 py-2 pl-2 pr-2
+            />
+          </div>
+          <div className="mx-4 mb-1">
+            <Button
+              onClick={this.props.onSubmit}
+              primary
+              fluid
+            >
+              Submit
```

Comment:
What about the Skip state?

Reply:
Di thought that `Submit` might be confusing for users, and instead suggested `Done` (static value) for the button. I've updated to this wording.
In addition, there's no additional step to 'skip' to - this is the final screen before the error report is submitted.